setup.py

```diff
@@ -32,7 +32,6 @@
     name=PROJECT_NAME,
     version=VERSION,
     url='https://github.com/Drakkar-Software/OctoBot-Commons',
-    license='LGPL-3.0',
```

Comment:
I think this line shouldn't be deleted.

Reply:
@Steap

Reply:
It could stay, but is not really required. The [documentation](https://packaging.python.org/guides/distributing-packages-using-setuptools/#license) states:

"The license argument doesn’t have to indicate the license under which your package is being released, although you may optionally do so if you want. If you’re using a standard, well-known license, then your main indication can and should be via the classifiers argument".